.github/workflows/gh-pages.yml

```diff
@@ -0,0 +1,60 @@
+# Sample workflow for building and deploying a Jekyll site to GitHub Pages
+name: Deploy Jekyll with GitHub Pages dependencies preinstalled
+
+on:
+  # Runs on pushes targeting the default branch
+  push:
+    branches: ["main"]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# Sets permissions of the GITHUB_TOKEN to allow deployment to GitHub Pages
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+# Allow only one concurrent deployment, skipping runs queued between the run in-progress and latest queued.
+# However, do NOT cancel in-progress runs as we want to allow these production deployments to complete.
+concurrency:
+  group: "pages"
+  cancel-in-progress: false
+
+jobs:
+  # Build job
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Setup Pages
+        uses: actions/configure-pages@v5
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 22
+      - run: |
+          npm install tailwindcss @tailwindcss/cli
+          npx @tailwindcss/cli -i ./readme/tailwind.src.css -o ./readme/tailwind.build.css
+      - name: Copy current example site
+        run: |
+          cp ./lib/feature_map/private/docs/index.html ./readme/example-docs-site.html
```

Comment:
Awesome! So this means each time changes are merged to main, the Github pages site will be updated AND a new version of the example docs site will be generated from the latest code to ensure it remains up to date. This is great!